src/linear_algebra/pi.lean

```diff
@@ -164,7 +164,10 @@ begin
     have : j ∉ I := assume hjI, hd ⟨hjI, hjJ⟩,
     rw [dif_neg this, zero_apply] },
   { simp only [pi_comp, comp_assoc, subtype_comp_cod_restrict, proj_pi, dif_pos, subtype.coe_prop],
-    ext b ⟨j, hj⟩, refl },
+    ext b ⟨j, hj⟩,
+    simp only [dif_pos, function.comp_app, function.eval_apply, linear_map.cod_restrict_apply,
```

Comment:
It looks to me like the `dif_pow` in the existing `simp only` above is perhaps no longer firing?

Reply:
Indeed, I just removed it.